sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -1,11 +1,47 @@
+        public System.DateTimeOffset Timestamp { get { throw null; } }
+        public System.Collections.Generic.ICollection<object> Values { get { throw null; } }
+        public void Add(System.Collections.Generic.KeyValuePair<string, object> item) { }
+        public void Add(string key, object value) { }
+        public void Clear() { }
+        public bool Contains(System.Collections.Generic.KeyValuePair<string, object> item) { throw null; }
+        public bool ContainsKey(string key) { throw null; }
+        public void CopyTo(System.Collections.Generic.KeyValuePair<string, object>[] array, int arrayIndex) { }
+        public byte[] GetBinary(string key) { throw null; }
+        public bool GetBoolean(string key) { throw null; }
+        public System.DateTime GetDateTime(string key) { throw null; }
+        public double GetDouble(string key) { throw null; }
+        public System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, object>> GetEnumerator() { throw null; }
+        public System.Guid GetGuid(string key) { throw null; }
+        public int GetInt32(string key) { throw null; }
+        public long GetInt64(string key) { throw null; }
+        public string GetString(string key) { throw null; }
+        public bool Remove(System.Collections.Generic.KeyValuePair<string, object> item) { throw null; }
+        public bool Remove(string key) { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+        public bool TryGetValue(string key, out object value) { throw null; }
+    }
     public partial class TableClient
     {
         protected TableClient() { }
         public virtual Azure.Response<Azure.Data.Tables.Models.TableItem> Create(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Data.Tables.Models.TableItem>> CreateAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Data.Tables.DynamicTableEntity> CreateEntity(Azure.Data.Tables.DynamicTableEntity entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<System.Collections.ObjectModel.ReadOnlyDictionary<string, object>> CreateEntity(System.Collections.Generic.IDictionary<string, object> entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Data.Tables.DynamicTableEntity>> CreateEntityAsync(Azure.Data.Tables.DynamicTableEntity entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
It looks like we've only updated the async version?

Reply:
Oh, I think I must have forgotten to export the API after I made the changes, and the checks went through because of inheritance.

Reply:
Oh, sorry, I guess I got confused why we still have the `IDictionary` overloads then.  Do we need them?  Folks could just create a new `DynamicTableEntity` from an existing dictionary.

Reply:
The plan was to iterate through full implementation of `DynamicTableEntity` typed methods until we have full coverage (including query expression validation), and then remove all the `IDictionary` typed methods.